src/lib.rs

```diff
@@ -117,7 +118,7 @@ pub trait Field:
 pub trait PrimeField: Field + From<u64> {
     /// The prime field can be converted back and forth into this binary
     /// representation.
-    type Repr: Default + AsRef<[u8]> + AsMut<[u8]>;
+    type Repr: Copy + Default + Send + Sync + AsRef<[u8]> + AsMut<[u8]>;
```

Comment:
Perhaps `'static` as well? Just to make sure there are no internal references and therefore the receiver can hold onto the type as long as they like...

Reply:
Yeah, that makes sense.